presto-product-tests/src/main/java/io/prestosql/tests/hive/TestGrantRevoke.java

```diff
@@ -231,6 +234,18 @@ public void testViewOwnerPrivileges()
         });
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testViewOwnerPrivilegesWithHiveOnlyViews()
+    {
+        executeWith(createViewAs("hive_only_view", format("SELECT * FROM %s", tableName), onHive()), view -> {
+            assertThatThrownBy(() -> onPresto().executeQuery(format("SELECT * FROM %s", view.getName())))
+                    .hasMessageContaining("Hive views are not supported");
+
+            assertThat(onPresto().executeQuery("SHOW GRANTS"))
+                    .doesNotHave(valueOnColumn(7, view.getName()));
```

Comment:
do you need to assert that? Maybe making sure query is passing is enough.

Reply:
- Would be good to check that `SHOW GRANTS` still includes _some_ results.
- how can we know that `7` is the right number? 
  if you did `onPresto().executeQuery("SHOW GRANTS").column(7)` and then `assertThan` on it both with positive (to prove this is the right column) and negative (to prove the view is excluded) assertions, it would be clear.
- please also add a test reading from `table_privilges` directly.

Reply:
It's ok, I will take it :)